docs/cli.md

```diff
@@ -34,12 +34,21 @@ Use "fuseml [command] --help" for more information about a command.
 
 Most CLI commands require you to supply the URL where the FuseML instance is running, either as a command line argument,
 or as the FUSEML_SERVER_URL environment variable. The URL is printed out by the installer during the installation process.
-If you missed it, you can retrieve it at any time with the following command:
+If you missed it, you can retrieve it at  any time with one of the following commands, depending on which type of ingress
+was used during installation (Traefik or Istio):
```

Comment:
Given the fact that there's no working example of fuseml with traefik, I would not mention it in the docs

Reply:
I want to agree, I don't like this any more than you do, but there's one case that doesn't fit this theory: k3s, which comes with Traefik preinstalled. In that case, our installer will default to using Traefic to provide ingresses for fuseml components (i.e. instead of installing Istio in addition to Traefik).
If it's true that there's no working example of fuseml with traefik, that means that fuseml is unusable with k3s.

Here are our options:
1. don't document k3s as a supported k8s distro
2. install istio as a mandatory component, even if/when traefik is present
3. keep traefik as a supported ingress solution for fuseml services and use it if already installed, and require istio to be installed as a prerequisite of kfserving for our current end-to-end example

We can do 2. easily.

For 3., we also need to modify the KNative installation script slightly to "detect" the istio managed domain, same as the installation binary does. With 3. implemented, we would have the following installation sequence on k3s:
* fuseml-installer defaults to traefik and uses that as an ingress for base fuseml components
* user needs to run `make kfserving-install` as a prerequisite to run our e2e example. This installs istio _in addition to Traefik_ and uses istio only for KNative/KFServing services.

I lean towards option 3. Option 2. establishes Istio as a prerequisite for FuseML, which is really not true: it's only a pre-requisite for KFServing, and KFServing is a 3rd party tool not mandatory for FuseML. Both 2. and 3., end up with 2 ingresses and 2 load balancers in your k8s cluster, so we may as well make it a post-installation, optional change.

Reply:
I'm not saying traefik is bad. Hopefully we will be able to come up with examples not using kfserving as serving component.

I'm just saying, as we do not have it now, we might not want to advertise traefik in our docs.

Reply:
I took your recommendation (we can call that option 4.): document that Traefik is not recommended and should not be installed as the default ingress for k3s installations.

With https://github.com/fuseml/fuseml/pull/139 and https://github.com/fuseml/fuseml/pull/137 in place, we can also postpone the installation of KFServing/KNative to after FuseML is installed, which doesn't require us to document it as a FuseML prerequisite anymore.

Given all that, I removed Traefik as an option for extracting the FuseML server URL, in the idea that we do not recommend using it as an ingress anymore, even though we do support k3s with Traefik preinstalled now (also thanks to https://github.com/fuseml/fuseml/pull/138 and https://github.com/fuseml/fuseml/pull/137).